### DISCUSSION TITLE
Menu Pagination

### DISCUSSION THREAD

**Discussion by mwakatumbula** [spesohq/laravel-ussd#16] [Q&A] [answered]
Hello Sparors
i had a quick question in how can I use menu paginateListing

**Comment by Tumi-D** [comment, marked as answer]
It doestnt seem to work on NOn GSM phones is  there a fix to it ?

**Reply by papamarfo** [reply]
Which country and phone are you using??

**Reply by Tumi-D** [reply]
In Ghana. Doesn't work on smart phones works on GSM (Yam)phones .

**Reply by papamarfo** [reply]
Which USSD service provider are you using??
